### PR TITLE
T1027 - Traduzir todo o Módulo de Receber e Pagar

### PR DESCRIPTION
**Pull request by rodrigets**
# Description

Atualiza tradução dos módulos:
- br_account
- br_account_payment

Obs: Tradução da tela completa inclui o PR https://github.com/multidadosti-erp/multidadosti-financial-addons/pull/114
## Type of change

Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

# How Has This Been Tested?

image

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: br_account_payment/i18n/pt_BR.po [hunk 1]
```diff
@@ -200,17 +205,17 @@ msgid "Name"
 msgstr "Nome"
 
 #. module: br_account_payment
-#: model:ir.model.fields,field_description:br_account_payment.field_account_move_alter_date_new_date_maturity
+#: model:ir.model.fields,field_descriptmultidadosti
```
Comment (mstuttgart) [inline br_account_payment/i18n/pt_BR.po]: @rodrigets o nome deste campo esta correto?

File: br_account_payment/i18n/pt_BR.po [hunk 2]
```diff
@@ -200,17 +205,17 @@ msgid "Name"
...
 msgid "New Date Maturity"
 msgstr "Nova Data de Vencimento"
 
 #. module: br_account_payment
-#: model:ir.model.fields,field_description:br_account_payment.field_account_move_alter_operation_new_financial_operation
+#: model:ir.model.fields,field_descriptmultidadostil_operation
```
Comment (mstuttgart) [inline br_account_payment/i18n/pt_BR.po]: @rodrigets o nome deste campo esta correto?

File: br_account_payment/i18n/pt_BR.po [hunk 3]
```diff
@@ -200,17 +205,17 @@ msgid "Name"
...
 msgid "New Financial Operation"
 msgstr "Nova Operação Financeira"
 
 #. module: br_account_payment
-#: model:ir.model.fields,field_description:br_account_payment.field_account_move_alter_operation_new_payment_mode
+#: model:ir.model.fields,field_descriptmultidadostimode
```
Comment (mstuttgart) [inline br_account_payment/i18n/pt_BR.po]: @rodrigets o nome deste campo esta correto?

Comment (mstuttgart) [inline br_account_payment/i18n/pt_BR.po]: @rodrigets o nome deste campo esta correto?

File: br_account_payment/i18n/pt_BR.po (hunk 1 above)
Comment (mstuttgart) [inline br_account_payment/i18n/pt_BR.po]: @rodrigets o nome deste campo esta correto?

File: br_account_payment/i18n/pt_BR.po (hunk 2 above)
Comment (mstuttgart) [inline br_account_payment/i18n/pt_BR.po]: @rodrigets o nome deste campo esta correto?

**Comment by rodrigets** [comment]
@mstuttgart , refiz a tradução , acredito que esteja correto agora.

**Pull request merged**